common/environment.go

```diff
@@ -209,6 +210,13 @@ func (EnvironmentVariable) AwsSessionToken() EnvironmentVariable {
 	return EnvironmentVariable{Name: "AWS_SESSION_TOKEN"}
 }
 
+func (EnvironmentVariable) GoogleAppCredentials() EnvironmentVariable {
+	return EnvironmentVariable{
+		Name: "GOOGLE_APPLICATION_CREDENTIALS",
```

Comment:
>GOOGLE_APPLICATION_CREDENTIALS [](start = 9, length = 30)

In the description, it was mentioned that the value should be "path/to/serviceAccountkey.json", does this mean the account credentials are expected to be provided unencrypted and stored on the disk? This approach wouldn't be very secure, I'd suggest to take in the credentials the same way as S3.

Reply:
Did we resolve this?